src/SearchEngine/MASA.BuildingBlocks.SearchEngine.AutoComplete/Options/AutoCompleteOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace MASA.BuildingBlocks.SearchEngine.AutoComplete.Options;
+public class AutoCompleteOptions
+{
+    public string Field { get; set; }
+
+    private int _pageIndex;
+
+    public int PageIndex
+    {
+        get => _pageIndex;
+        set
+        {
+            if (value <= 0)
+                throw new ArgumentException($"{nameof(PageIndex)} must be greater than or equal to 1", nameof(PageIndex));
```

Comment:
geater than 0?

Reply:
You are right, the PageIndex here is the page number, but it can easily be thought of as the index